components/dashboards-web-component/src/auth/Login.jsx

```diff
@@ -176,6 +179,33 @@ export default class Login extends Component {
                             label={<FormattedMessage id="login.title" defaultMessage="Login"/>}
                             disabledBackgroundColor="rgb(27, 40, 47)"
                         />
+                        <br />
+                        <br />
+                        <div style={styles.cookiePolicy}>
+                            <div>
+                                After a successful sign in, we use a cookie in your browser to track your session.&nbsp;
```

Comment:
Shouldn't this be internalized? 
We have added i18n support for dashboard, right? If so, then these texts should be internalized too.